### ISSUE TITLE
"export 'extras' was not found in 'mobx'

### ISSUE THREAD

**Issue by simspace-dr** [mobxjs/mobx-react#449]
I am using yarn and trying to upgrade mobx to v4.1.1 and mobx-react to v4.4.3. 

Yarn is giving me this warning:
`warning " > mobx-react@4.4.3" has incorrect peer dependency "mobx@^2.6.3 || ^3.0.0".`

I ran `yarn remove mobx mobx-react` and when that was completed, I deleted the yarn.lock file and confirmed mobx and mobx-react were no longer in `./node_module`. 

Then I added the new versions to my package.json ...
```
    "mobx": "^4.1.1",
    "mobx-react": "^4.4.3",
```
But when I run `yarn` again,  I have the same problem, and when I look at  `./node_module/mobx-react/package.json` I find this ...
```
  "peerDependencies": {
    "mobx": "^2.6.3 || ^3.0.0",
    "react": "^0.13.0 || ^0.14.0 || ^15.0.0 || ^16.0.0"
  },
``` 

Any suggestions how I can solve this problem, so I can get to the latest mobx and mobx-react versions?
Thanks!
Chris

**Comment by mweststrate** [comment]
You need mobx-react 5, not 4

Op wo 4 apr. 2018 00:17 schreef Chris Simeone <notifications@github.com>:

**Comment by simspace-dr** [comment]
I didn't realize there was a version 5. When I go to [mobx-react on Github](https://github.com/mobxjs/mobx-react), the last version I see is [4.4.3 in the change log](https://github.com/mobxjs/mobx-react/blob/master/CHANGELOG.md#443). 

I'll change my `package.json` and give that a try. 

Is version 5 a beta or final release?

**Comment by mweststrate** [comment]
final, will update the changelog, I think that was an oversight

**Comment by moonlitusun** [comment]
mobx-react 5 is not support some brower,what can i fixed #449

**Comment by mweststrate** [comment]
@moonlitusun please open new issues for new issues. Is your problem same as #608 ?
